### DISCUSSION TITLE
Missing settings.json.tmp file.

### DISCUSSION THREAD

**Discussion by Legotron123** [X2CommunityCore/xcom2-launcher#298] [Help] [answered]
Just downloaded this launcher and some mods today, and when I tried launching the game, it said the pathway to settings.json.tmp was denied and none of the mods would work. I looked and my computer said the pathway it was trying to use didn't exist, so I'm assuming I'm missing a file? What do I do?

**Comment by Legotron123** [comment, marked as answer]
Nevermind, I'm just an idiot who didn't have them selected. Still don't have the settings file though.

**Reply by RevZero** [reply]
`settings.json.tmp` is only a backup file is created every time settings are saved. It should be created after you close AML for the first time, for example.

**Reply by Legotron123** [reply]
For some reason it's not doing that though. I get the same pop up when I close AML.

**Reply by RevZero** [reply]
Ah OK. In what folder is your AML copy located? Keep it out of system folders and try to have it in a folder on your desktop. 
If it still complains, maybe some Antivirus tool is preventing access, so temporarily disable any Antivir you may have and see if that helps.
Last but not least you could try the latest beta version: [XCOM2_AlternativeModLauncher_1.5.0-beta2.zip](https://github.com/X2CommunityCore/xcom2-launcher/releases/download/v1.5.0-beta.2/XCOM2_AlternativeModLauncher_1.5.0-beta2.zip)

**Reply by Legotron123** [reply]
That did it. Moved it from program files to a generic other folder. Thanks for your help!
